src/main/resources/META-INF/resources/webjars/service/prov/prov.css

```diff
@@ -483,3 +483,7 @@ img.flag-icon.prov-location-flag {
 #instance-ram-unit {
   width: 93px;
 }
+
+#subscribe-configuration-prov .advanced{
```

Comment:
The logic seems to be inverted.
The class should work like this :
#subscribe-configuration-prov .modal:not(.advanced) .element-advanced {display:none !important}
and the JS should on toggle the advanced class on the modal